packages/plugin-core/src/workspace/tutorialInitializer.ts

```diff
@@ -16,7 +15,6 @@ import {
 } from "@dendronhq/engine-server";
 import fs from "fs-extra";
 import path from "path";
```

Comment:
can we also replace `rif` from our dependencies now? (I don't think we're using it anywhere)

I believe you can just run `yarn remove {pkg}` inside the workspace